src/javascripts/todo-filter.js

```diff
@@ -39,4 +39,4 @@ function updateFilter (filter) {
   }
 }
 
-export default prepFilters
+module.exports = prepFilters
```

Comment:
module.exports is part of the CommonJS syntax. It is considered a bad idea to mix ES6 module syntax (like import or export) and the CommonJS syntax (like require and module.exports). This is not an acceptable solution.